docs/contracts.rst

```diff
@@ -976,8 +975,7 @@ are all compiled as calls (``DELEGATECALL``) to an external
 contract/library. If you use libraries, take care that an
 actual external function call is performed.
 ``msg.sender``, ``msg.value`` and ``this`` will retain their values
-in this call, though (prior to Homestead, ``msg.sender`` and
-``msg.value`` changed, though).
```

Comment:
Actually I think we should extend this with an explanation: "because of the use of ``CALLCODE`` and not ``DELEGATECALL``"